npm-shrinkwrap.json

```diff
@@ -90,892 +88,821 @@
-        "@aws-sdk/middleware-host-header": "3.40.0",
-        "@aws-sdk/middleware-logger": "3.40.0",
-        "@aws-sdk/middleware-retry": "3.40.0",
-        "@aws-sdk/middleware-serde": "3.40.0",
-        "@aws-sdk/middleware-stack": "3.40.0",
-        "@aws-sdk/middleware-user-agent": "3.40.0",
-        "@aws-sdk/node-config-provider": "3.40.0",
-        "@aws-sdk/node-http-handler": "3.40.0",
-        "@aws-sdk/protocol-http": "3.40.0",
-        "@aws-sdk/smithy-client": "3.41.0",
-        "@aws-sdk/types": "3.40.0",
-        "@aws-sdk/url-parser": "3.40.0",
-        "@aws-sdk/util-base64-browser": "3.37.0",
-        "@aws-sdk/util-base64-node": "3.37.0",
-        "@aws-sdk/util-body-length-browser": "3.37.0",
-        "@aws-sdk/util-body-length-node": "3.37.0",
-        "@aws-sdk/util-user-agent-browser": "3.40.0",
-        "@aws-sdk/util-user-agent-node": "3.40.0",
-        "@aws-sdk/util-utf8-browser": "3.37.0",
-        "@aws-sdk/util-utf8-node": "3.37.0",
-        "tslib": "^2.3.0"
-      },
-      "dependencies": {
-        "tslib": {
-          "version": "2.3.1",
-          "resolved": "https://registry.npmjs.org/tslib/-/tslib-2.3.1.tgz",
-          "integrity": "sha512-77EbyPPpMz+FRFRuAFlWMtmgUWGe9UOG2Z25NqCwiIjRhOf5iKGuzSe5P2w1laq+FkRy4p+PCuVkJSGkzTEKVw=="
-        }
+      "version": "3.10.0",
+      "resolved": "https://registry.npmjs.org/@aws-sdk/client-sso/-/client-sso-3.10.0.tgz",
+      "integrity": "sha512-IxUtpxky1Kadd10cTyFHfwyEWyBra/ckN72JXiafYogrvQILju0pkOwD0+inttXzVqTmDFKGDpUM/fHfHdmASA==",
+      "requires": {
+        "@aws-crypto/sha256-browser": "^1.0.0",
+        "@aws-crypto/sha256-js": "^1.0.0",
+        "@aws-sdk/config-resolver": "3.10.0",
+        "@aws-sdk/fetch-http-handler": "3.10.0",
+        "@aws-sdk/hash-node": "3.10.0",
+        "@aws-sdk/invalid-dependency": "3.10.0",
+        "@aws-sdk/middleware-content-length": "3.10.0",
+        "@aws-sdk/middleware-host-header": "3.10.0",
+        "@aws-sdk/middleware-logger": "3.10.0",
+        "@aws-sdk/middleware-retry": "3.10.0",
+        "@aws-sdk/middleware-serde": "3.10.0",
+        "@aws-sdk/middleware-stack": "3.10.0",
+        "@aws-sdk/middleware-user-agent": "3.10.0",
+        "@aws-sdk/node-config-provider": "3.10.0",
+        "@aws-sdk/node-http-handler": "3.10.0",
+        "@aws-sdk/protocol-http": "3.10.0",
+        "@aws-sdk/smithy-client": "3.10.0",
+        "@aws-sdk/types": "3.10.0",
+        "@aws-sdk/url-parser": "3.10.0",
+        "@aws-sdk/url-parser-native": "3.10.0",
+        "@aws-sdk/util-base64-browser": "3.10.0",
+        "@aws-sdk/util-base64-node": "3.10.0",
+        "@aws-sdk/util-body-length-browser": "3.10.0",
+        "@aws-sdk/util-body-length-node": "3.10.0",
+        "@aws-sdk/util-user-agent-browser": "3.10.0",
+        "@aws-sdk/util-user-agent-node": "3.10.0",
+        "@aws-sdk/util-utf8-browser": "3.10.0",
+        "@aws-sdk/util-utf8-node": "3.10.0",
+        "tslib": "^2.0.0"
```

Comment:
These and several other modules are being downgraded in this PR. But even after running the `git` and `npm` commands that I mention in another comment, this still happens. I don't understand why. 🤔  So far I was not able to pinpoint a (transitive) dependency that requires the downgrade of all these packages.